corehq/apps/reminders/forms.py

```diff
@@ -723,7 +723,7 @@ def section_recipient(self):
                     InlineField(
                         'recipient_case_match_property',
                         css_class="input-xlarge",
-                        data_bind="typeahead: getAvailableSubcaseProperties",
+                        data_bind="autocompleteSelect2: getAvailableSubcaseProperties,",
```

Comment:
here and one below as well. I'm not sure if it breaks anything or not, just flagging.